bindings/python/setup.py

```diff
@@ -25,13 +25,14 @@ def patched_copyxattr(src, dst, *, follow_symlinks=True):
 project_dir = os.path.abspath(os.path.join(os.path.join(root_dir, "..", "..")))
 sdist_dir = os.path.join(os.path.join(root_dir, "mata"))
 src_dir = sdist_dir if os.path.exists(sdist_dir) else project_dir
+mata_build_dir = "build-python"
```

Comment:
I'm still baffled, why you cannot to this with original `build` dir and why do you need some special build directory?

Reply:
The discussion is continued in https://github.com/VeriFIT/mata/pull/315#discussion_r1312615343.